pages/index.js

```diff
@@ -111,13 +95,13 @@ export default function HomePage() {
     }
     response.json();
     setEditingTransaction(null);
-    setIsFormVisible(false);
+    setIsFormOpen(false);
     await mutate();
```

Comment:
```suggestion
   mutate();
```
Since you don't wait for anything you can get rid of it. Same in line 121.